source/BatchedGeometry.h

```diff
@@ -0,0 +1,195 @@
+/*-------------------------------------------------------------------------------------

```

Comment:
how did the header end up here?

Reply:
First time using a source control system like GitHub.  I copied all files that I edited into the source directory of my pull request.  I removed it from the branch and then uploaded it to the include folder.